src/frontend/src/meta_client.rs

```diff
@@ -246,11 +249,15 @@ impl FrontendMetaClient for FrontendMetaClientImpl {
         self.0.list_hummock_meta_config().await
     }
 
-    async fn list_compact_task_assignment(&self) -> Result<Vec<CompactTaskAssignment>> {
+    async fn list_compaction_status(&self) -> Result<Vec<CompactTaskAssignment>> {
```

Comment:
Given that the RPC is ListCompactTaskAssignment and the returning type is CompactTaskAssignment, I think it is better to keep the original function name.